jakarta.ls/src/main/resources/transactions.json

```diff
@@ -1,3 +1,14 @@
 {
-
-}
\ No newline at end of file
+    "@Transactional": {
+      "prefix": "transactionbehav",
+      "body": [
+        "@Transactional(",
+        "\trollbackOn={SQLException.class, IOException.class},",
+        "\tdontRollbackOn={IllegalStateException.class})"
+      ],
+      "description": "Specify exception types that will and will not cause a rollback for transaction",
```

Comment:
Update description to: "Transactional annotation - specify exception types that will and will not cause a rollback for transaction